Add pt-BR translation

madcampos opened the pull request:
Added brazilian portuguese translation (based on the `de` and `pt-PT` files) and fixed a typo for pt-PT.

madcampos committed:
Add pt-BR translation

Added brazilian portuguese translation (based on the `de` and `pt-PT` files) and fixed a typo for pt-PT.

Boldewyn commented:
Cool, thank you! Two minor questions, before I merge it:
- Good catch with the typo you spotted! Judging from the other strings around, should `Introduza` be lower-case, too?
- Could the additional translations in `src/pt-BR.json` be used for `pt-PT` as well, or would they be different in Portugal?

madcampos committed:
Fix typo and add new translations to pt-PT.

The translations are based in the brazilian portuguese version so there might be inconsistencies.

madcampos commented:
Addressed the typo, indeed it should be lower-case.
Also added the translations based on the `pt-BR` translation. It should be noted that there are differences and the translation could differ, I changed the one that I know/spotted:
`file` => `arquivo` in `pt-BR` and `ficheiro` in `pt-PT`
Still, there can be other differences.
